d2cli/help.go

```diff
@@ -38,6 +39,7 @@ Subcommands:
   %[1]s layout [name] - Display long help for a particular layout engine, including its configuration options
   %[1]s themes - Lists available themes
   %[1]s fmt file.d2 ... - Format passed files
+  %[1]s play file.d2 - Opens the file in playground
```

Comment:
`playground` might be unknown to folks. I'd add a description like `Opens the file in the d2 playground, an online web viewer`